src/db.c

```diff
@@ -2116,10 +2128,10 @@ int georadiusGetKeys(struct redisCommand *cmd, robj **argv, int argc, getKeysRes
 
     /* Add all key positions to keys[] */
     keys[0].pos = 1;
-    keys[0].flags = CMD_KEY_WRITE;
+    keys[0].flags = 0;
```

Comment:
maybe add a top comment for `georadiusGetKeys` about the fact it doesn't maintain flags and why